src/components/moderation/modmail.ts

```diff
@@ -40,6 +44,29 @@ export default class Modmail extends BotComponent {
         component_state: moderation_state;
     }>();
 
+    override async setup(commands: CommandSetBuilder) {
+        commands.add(
+            new TextBasedCommandBuilder("wsetupmodmailsystem", EarlyReplyMode.none)
+                .set_permissions(Discord.PermissionFlagsBits.Administrator)
+                .set_description("Create modmail message here")
+                .set_handler(this.modmail_setup.bind(this)),
+        );
+        commands.add(
+            new MessageContextMenuInteractionBuilder("Update modmail message")
```

Comment:
I'm a bit concerned about doing this because it'll show up on the context menu for every message on the server

Reply:
Yeah, I was also a bit worried about that. But I thought it'll only show up for admins anyways, and would only work on a wheatley message. But you're probably right, I'll make it a normal command.